projects/fluid_simulation/sim.cpp

```diff
@@ -395,8 +403,14 @@ void FluidSimulationApp::Render()
 
         // Prepare the image to present.
         ScalarInput si;
-        si.texelSize   = ppx::float2(1.0f / GetWindowWidth(), 1.0f / GetWindowHeight());
-        si.ditherScale = mDitheringGrid->GetDitherScale(GetWindowWidth(), GetWindowHeight());
+        si.texelSize     = ppx::float2(1.0f / GetWindowWidth(), 1.0f / GetWindowHeight());
+        si.ditherScale   = mDitheringGrid->GetDitherScale(GetWindowWidth(), GetWindowHeight());
+        if (GetConfig().pEnableBloom->GetValue())
+            si.filterOptions |= kDisplayBloom;
```

Comment:
Would you mind putting {} around those for consistency and to prevent potential bugs in the future?

Reply:
Oops.  Done.